src/lib/y2packager/dialogs/product_license_translations.rb

```diff
@@ -0,0 +1,106 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2018 SUSE LLC, All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm"
+require "cwm/dialog"
+require "y2packager/widgets/product_license_translations"
+
+Yast.import "Label"
+
+module Y2Packager
+  module Dialogs
+    # This dialog displays available translation for a given product.
+    #
+    # The dialog is open as a pop-up (check #should_open_dialog,
+    # #wizard_create_dialog and #layout for technical details) and it relies
+    # heavily on the {Y2Packager::Widgets::ProductLicenseTranslations} widget.
+    class ProductLicenseTranslations < CWM::Dialog
+      # @return [Y2Packager::Product] Product
+      attr_reader :product
+      # @return [String] Default language code (eg. "en_US")
+      attr_reader :language
+
+      def initialize(product, language = nil)
```

Comment:
NP: Probably it is pretty clear, but I would document the type of the parameters.